pkgs/development/compilers/dtc/fixstatic.patch

```diff
@@ -0,0 +1,30 @@
+diff -ur dtc-before/libfdt/meson.build dtc-after/libfdt/meson.build
+--- dtc-before/libfdt/meson.build	2024-05-07 15:58:14.070792700 -0300
++++ dtc-after/libfdt/meson.build	2024-05-07 15:58:46.543032425 -0300
+@@ -27,7 +27,7 @@
+ 
+ link_args += version_script
+ libfdt = library(
+-  'fdt', sources,
++  'fdt-so', sources,
```

Comment:
Why is this needed? This breaks static builds for QEMU for me (though admittedly static builds for QEMU require other patches not yet on master anyway).